### PR TITLE
Announce wasm CSP change in MV3

### PR DESCRIPTION
**Pull request by dotproto**

### CODE REVIEW

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: site/en/docs/extensions/whatsnew/index.md [hunk 1]
```diff
@@ -22,6 +22,14 @@ May 26, 2022
 Added the [trader/non-trader developer identification](/docs/webstore/trader-disclosure) that
 informs developers to accurately self-declare their trader/non-trader status.
 
+### Chrome 103: Wasm in Manifest V3 requires wasm-unsafe-eval {: #m103-wasm-csp }
```
[the remark this answers is not in the document]
Comment (dotproto) [reply site/en/docs/extensions/whatsnew/index.md:25]: We've been pretty consistent about using "Manifest V3". I think most of the matches you're seeing when searching for "v3" are coming from paths in URLs. I just searched for a few terms in the `site/en/docs/extensions/` directory of my local branch and gathered the results

Search | matches
--- | ---
`manifest v3` | 0
`Manifest v3` | 0
`manifest V3` | 0
`Manifest V3` | 143

**Review by jpmedley — APPROVED** [review]

**Comment by sofiayem** [comment]
@dotproto @jpmedley Is this blocked or should we merge?

**Comment by jpmedley** [comment]
It's @dotproto 's call.

**Comment by dotproto** [comment]
Oops, accidentally overwrote this branch because I thought it was already merged. Nothing was blocking this, I just lost track of the PR 😅

**Pull request merged**